vernac/metasyntax.ml

```diff
@@ -504,27 +507,53 @@ let find_prod_list_loc sfmt fmt =
     (* A separator; we highlight the separating sequence *)
     Loc.merge_opt (fst (List.hd sfmt)) (fst (List.last sfmt))
 
+let is_blank s =
+  let n = String.length s in
+  let rec aux i s = i > n || s.[0] = ' ' && aux (i+1) s in
+  aux 0 s
```

Comment:
This is wrong and should read instead:
```ocaml
let rec aux i s = i >= n || (s.[i] = ' ' && aux (i + 1) s) in
```

Reply:
Right. I think it would have been faster that you directly suggest this variant, rather than me quickly doing it without really having time to concentrate on and double-check what I'm doing because doing it in the middle of an already busy schedule, but feeling obliged to do it now because I'd also like that some progresses are eventually made in the PRs (and implicitly counting on you anyway to check).

Should I have actually asked you to do it from the beginning rather than quickly doing it myself?

Reply:
This conversation is taking more time than me having to write off the code, so in the end the answer is yes, but this was hard to forecast initially...

Reply:
> This conversation is taking more time than me having to write off the code.

Indeed, and if I'm pushing us to a meta-level discussion, it is that our way of working involving context switching (at unplanned time) is (at least for me) psychologically exhausting and leads to wasting time and energy. Something has to change!

As for using the suggestion feature, yes, feel free to do it on my PRs.

Reply:
I think that the PR notification system is very bad as it incentivizes us to react immediately, but this should not be the case. There is this incomplete "left for later", or whatever its name is, tag system but it is very impractical. My stance on this is that we shouldn't answer directly to requests for change, only keeping an eye from time to time on the lists of things we have to perform, or when explicitly asked by a ping by a human being.

Reply:
What is this "left for later" tag system?

> My stance on this is that we shouldn't answer directly to requests for change, only keeping an eye from time to time on the lists of things we have to perform, or when explicitly asked by a ping by a human being.

Somehow, I agree with you and I will think at not answering too quickly to notifications (even if it may mean in practice to have to wait for the next period with low constraints, such as the next week-end).

Let me say also that I'm particularly worry these days by the amount of my PRs around printing/impargs now opened for months and whose absence of reviews is not good for working smoothly. What should we do: should we start considering that some of us merge themselves their PRs when they feel confident enough about them and noone has time to review? Should we use temporary forks of master so that we can continue working until reviews can be done (and the reviews are then done on the incremental PRs to the fork, and the fork is merged in one go)? Who has similar worries?

We should probably also have more collective "reviewing" sessions, slot of times where we are connected at the same time not to discuss plenarily but to address the things requiring interactions. The fact that such meeting is scheduled would allow to mentally prepare oneselves to a context switch. It would also allow help to address (exhausting) latencies.